tests/utils/actions/misc.ts

```diff
@@ -32,6 +32,8 @@ function shamefullyIgnoreConsoleMessage(message: ConsoleMessage): boolean {
     // Firefox warn on quill
     // See https://github.com/quilljs/quill/issues/2030
     '[JavaScript Warning: "Use of Mutation Events is deprecated. Use MutationObserver instead."',
+    // edgeless.spec.ts assert.ts › assertEdgelessPointColor
```

Comment:
It only occurs in testing. Because the test case reads canvas image data multiple times in a very short time. I optimize the test case to read image data in one operation.